### PR TITLE
add : added isloggedin for checking user

### PR DESCRIPTION
**Pull request by Sukomal07**
added isloggedin route for auth

### CODE REVIEW

**Comment by krishnaacharyaa** [comment]
add: karkke tag kahaa hei bhaaiii 😅

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
I've to go through everything once again once everything is merged to see if things are set up properly, and we need your session on this to understand this better. Let's have it today 7pm

File: backend/controllers/auth-controller.js [hunk 1]
```diff
@@ -401,3 +399,33 @@ export const signOutUser = asyncHandler(async (req, res) => {
       new ApiResponse(HTTP_STATUS.OK, '', RESPONSE_MESSAGES.USERS.SIGNED_OUT)
     )
 });
+
+// check user
+export const isLoggedIn = asyncHandler(async (req, res) => {
+  const { _id } = req.params
```
[the remark this answers is not in the document]
Comment (krishnaacharyaa) [reply backend/controllers/auth-controller.js:405]: And were are we actually refreshing the access token and refresh token?

[the remark this answers is not in the document]
Comment (krishnaacharyaa) [reply backend/controllers/auth-controller.js:405]: Yes lets do it that way. can you push that changed commit :) @Sukomal07

[the remark this answers is not in the document]
Comment (Sukomal07) [reply backend/controllers/auth-controller.js:405]: Yes

**Review by krishnaacharyaa — APPROVED** [review]

**Pull request merged**